frontend/src/components/pages/events/CountdownButton.tsx

```diff
@@ -31,12 +31,92 @@ interface Props {
   currentTime: string;
   isSignedUp: boolean;
   isOnWaitingList: boolean;
+  positionOnWaitinglist: number;
   isFull: boolean;
   loading: boolean;
   disabled?: boolean;
+  currentTimeParts: string[];
   onClick: () => void;
 }
 
+const ButtonText: React.FC<Props> = ({
+  currentTimeParts,
+  isSignedUp,
+  isOnWaitingList,
+  countDownDate,
+  currentTime,
+  isFull,
+  positionOnWaitinglist,
+}) => {
+  const translate = (timeWord: string, time: number) => {
+    if (timeWord === "days") return time > 1 ? "dager" : "dag";
+    if (timeWord === "hours") return time > 1 ? "timer" : "time";
+    if (timeWord === "minutes") return time > 1 ? "minutter" : "minutt";
+    if (timeWord === "seconds") return time > 1 ? "sekunder" : "sekund";
+  };
+
+  const [now, setNow] = useState(dayjs(currentTime));
+  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft(countDownDate, now));
```

Comment:
It appears that you moved the `now` and `timeLeft` states from `CountdownButton`, but forgot to move the `useEffect`:
```
useEffect(() => {
  // Update timeLeft and now (current time) each second
  const id = setTimeout(() => {
    setTimeLeft(calculateTimeLeft(countDownDate, now));
    setNow(now.add(1, "second"));
  }, 1000);
  return () => {
    clearTimeout(id);
  };
});
```
This means that `setNow` and `setTimeLeft` are never called, which is what causes the Vercel test deployment to fail.